content/ember-4-0-released.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Ember 4.0 released
+authors:
+  - jen-weber
+  - matthew-beale
+date: 2021-11-22T00:00:00.000Z
+tags:
+  - releases
+  - '2021'
+  - version-4-x
+---
+
+Today, the Ember project is releasing version 4.0 of Ember.js, Ember Data, and
+Ember CLI. This major version bump focuses
+the framework by removing long-deprecated APIs and support for legacy platforms.
+
+Although Ember "Octane" APIs have been the default for new applications since
+Ember 3.15, the framework has continued to support "Classic" framework features
+in accordance with our semantic versioning commitment. Ember 4.0 takes a step
+forward and drops already-deprecated classic APIs, however the foundational `EmberComponent`
+and `EmberObject`/`computed` APIs are not removed in this release.
+
+This blog post will help you understand how to upgrade and navigate any breaking changes.
+
+Additionally, today the 4.1 beta cycle begins for these same projects. Ember's
```

Comment:
I'd drop this whole paragraph. We don't always have all the betas out by the time of the release post and we usually consider the cycle starting when Ember.js itself branches. The info also seems superfluous to the post.